### ISSUE TITLE
VCDebugLauncher installs targets files into wrong location

### ISSUE THREAD

**Issue by huguesv** [Microsoft/PTVS#5248]
Files are installed into: `C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\VC\VCTargets\Platforms`

They need to be installed into: `C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Microsoft\VC\v160\Platforms`

Our vsix properties are looking correct, and the built .vsix as well, so it may be that we need to make changes  in setup .swr/.swixproj files.

**Comment by wjk** [comment]
This is a known issue with VS2019's VSIX support. I [reported it to Microsoft](https://developercommunity.visualstudio.com/content/problem/498941/vsix-vctargets-install-root-still-points-to-old-lo.html), but they said that they have no plans on updating the VCTargets install root to work in VS2019; we will have to change to use the MSBuild install root and prefixing the TargetPath with `Microsoft\VC\v160`. I'm going to prepare a PR for this soon. Hope this helps!

**Comment by huguesv** [comment]
@wjk I am unsure at this point, there are setup files in VS private repo as well as here.

**Comment by huguesv** [comment]
Log for when trying to install our VSIX into a VS with C++/Python workloads:
```
4/24/2019 4:40:43 PM - Capping recursion at 5 directories in each extension root due to 'default limit'.
4/24/2019 4:40:43 PM - Found 'C:\PROGRAM FILES (X86)\COMMON FILES\MICROSOFT\EXTENSIONMANAGER\EXTENSIONS\Microsoft\Windows Kits\10\Desktop SDK\extension.vsixmanifest'
4/24/2019 4:40:43 PM - Found 'C:\PROGRAM FILES (X86)\COMMON FILES\MICROSOFT\EXTENSIONMANAGER\EXTENSIONS\Microsoft\Windows Kits\10\SDK ARM Additions\extension.vsixmanifest'
4/24/2019 4:40:43 PM - Found 'C:\PROGRAM FILES (X86)\COMMON FILES\MICROSOFT\EXTENSIONMANAGER\EXTENSIONS\Microsoft\Windows Kits\10\Windows Store SDK\extension.vsixmanifest'
4/24/2019 4:40:43 PM - Skipping 'C:\Users\huvalo\AppData\Local\Microsoft SDKs' as it doesn't exist
4/24/2019 4:40:43 PM - Beginning to install extension to Visual Studio Enterprise 2019 d16.1stg...
4/24/2019 4:40:43 PM - VSIX TaskScheduler: VSIX Auto Updatetask does not exist.
4/24/2019 4:40:43 PM - VSIX TaskScheduler: VSIX Auto Updatetask has been successfully created.
4/24/2019 4:40:44 PM - Install Error : System.InvalidOperationException: The path '[installdir]\Common7\IDE\VC\VCTargets' defined in the manifest.json file is not valid.
   at Microsoft.VisualStudio.ExtensionManager.PackageInstaller.ValidatePackage(VsixPackage vsixPackage, String installDir, Boolean installPerMachine, String sourceFile, Version targetedVSVersion)
   at Microsoft.VisualStudio.ExtensionManager.PackageInstaller.Validate(ZipPackage packageContainer, IEngineHost engineHost)
   at Microsoft.VisualStudio.ExtensionManager.DefaultPackageInstaller.Validate(ZipPackage packageContainer, IEngineHost engineHost)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.PerformSetupEngineInstall(InstallableExtensionImpl extension, Boolean installPerMachine, Boolean isPackComponent, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, IProgress`1 progress, InstallFlags installFlags, AsyncOperation asyncOp, Version targetedVsVersion, IInstalledExtension& newExtension)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.InstallInternal(InstallableExtensionImpl extension, InstallFlags installFlags, IDictionary`2 extensionsInstalledSoFar, List`1 extensionsUninstalledSoFar, IInstalledExtensionList modifiedInstalledExtensionsList, AsyncOperation asyncOp, IProgress`1 progress, Version targetedVsVersion)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.BeginInstall(IInstallableExtension installableExtension, InstallFlags installFlags, AsyncOperation asyncOp, Version targetedVsVersion)
   at Microsoft.VisualStudio.ExtensionManager.ExtensionEngineImpl.InstallWorker(IInstallableExtension extension, InstallFlags installFlags, AsyncOperation asyncOp)
```

**Comment by huguesv** [comment]
The VSIX installer bug above has now been fixed, but wasn't the cause of this issue.

The short answer is that it's installing an older version of the VCLauncher, rather than the fixed one.

When installing the latest build, it is installing an older 18290.2 from Dec 2018 of VCLauncher:

Cache folder: Microsoft.PythonTools.Debugger.VCLauncher.Vsix,version=16.0.0
```
"url": "https:\/\/vsdrop.corp.microsoft.com\/file\/v1\/Products\/DevDiv\/Microsoft\/PTVS\/master\/18290.2;\/Microsoft.PythonTools.Debugger.VCLauncher.vsix",
```

While all the other components are correctly from build 19113.1

Cache folders:
```
Microsoft.PythonTools.Core.Vsix,version=16.0.19113.1
Microsoft.PythonTools.NativeDevelopment.Templates.Vsix,version=16.0.19113.1
...
```


The component group definition has these dependencies:
```
vs.dependencies
  vs.dependency id=Microsoft.Component.PythonTools
                version=[$(VSGeneralVersion),$(VSNextGeneralVersion))
  vs.dependency id=Microsoft.VisualStudio.Component.VC.CoreIDE
                version=[$(VSGeneralVersion),$(VSNextGeneralVersion))
  vs.dependency id=Microsoft.VisualStudio.Component.Windows10SDK
                version=[$(VSGeneralVersion),$(VSNextGeneralVersion))
  vs.dependency id=Microsoft.VisualStudio.Component.VC.DiagnosticTools
                version=[$(VSGeneralVersion),$(VSNextGeneralVersion))
  vs.dependency id=Microsoft.VisualStudio.Component.Windows10SDK.17763
                version=[$(VSGeneralVersion),$(VSNextGeneralVersion))
  vs.dependency id=Microsoft.PythonTools.NativeDevelopment.Templates.Vsix
                version=[$(VSGeneralVersionInsertedDependency),$(VSNextGeneralVersion))
  vs.dependency id=Microsoft.PythonTools.Debugger.VCLauncher.Vsix
                version=[$(VSGeneralVersionInsertedDependency),$(VSNextGeneralVersion))

```

**Comment by huguesv** [comment]
It's not bundling the latest version because the version of the vsix is fixed at 16.0.0, unlike the other vsixes we have that use the build number.
